pkg/cmd/version/version.go

```diff
@@ -0,0 +1,63 @@
+package version
+
+import (
+	"github.com/jenkins-x/jx/v2/pkg/version"
+
+	"github.com/jenkins-x/jx/v2/pkg/cmd/helper"
+
+	"github.com/jenkins-x/jx/v2/pkg/cmd/opts"
+	"github.com/jenkins-x/jx/v2/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+// Options struct contains the version flags
+type Options struct {
+	*opts.CommonOptions
+	short bool
+}
+
+// NewCmdVersion creates the version command
+func NewCmdVersion(commonOpts *opts.CommonOptions) *cobra.Command {
+	options := &Options{
+		CommonOptions: commonOpts,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "Print the version information",
+		Run: func(cmd *cobra.Command, args []string) {
+			options.Cmd = cmd
+			options.Args = args
+			err := options.Run()
+			helper.CheckErr(err)
+		},
+	}
+
+	cmd.Flags().BoolVar(&options.short, "short", false, "Print version")
```

Comment:
wonder whether it would make sense to have a `--machine-readable` flag or maybe more a `-o` flag which would be more used in other commands already. Right now the output of the version commands is:

```
$ jx version
Version        2.1.48-dev+3064c5b7c
Commit         3064c5b7c
Branch         jx-7081
Build date     20200527-20:10:18
Go version     1.13.11
Git tree state clean
```

resp

```
jx version --short
Version 2.1.48-dev+3064c5b7c
```

Sometimes for scripting, it is nice to have an easy way to just get to a specific piece of information without having to use _sed_, _awk_ or similar. So having the option to write the result as for example json could be handy. Alternatively, `-short` could just print the version without anything else. This way at least the version can be directly retrieved in a script. WDYT?

BTW, I like this change a lot. Well done!

Reply:
I was thinking along the same lines, but we should think about a list of commands which should have the -o flag (or may be make it global), and what format we plan to support (json, yaml, toml etc..). It should be a separate ticket imo (There will be lot of changes), thoughts?

Reply:
Another thing which comes to mind, I am wondering how useful the 'Branch' value is. While working on the codebase you would almost always be on a branch, in the pipeline, however, we are on a detached HEAD state. So I guess the branch name would become 'unkown'. This would, in particular, be the case for release builds which is probably not a good idea.

Reply:
One more - maybe we can format the build date a bit nicer - '20200527-20:10:18', something along the lines of `2020-04-06T16:33:17Z`. Some separators in the date at least.